flare/algorithm_zoo/ddpg.py

```diff
@@ -0,0 +1,168 @@
+from copy import deepcopy
+import math
+from flare.framework.algorithm import Algorithm, Model
+from flare.framework import common_functions as comf
+import torch
+import torch.nn as nn
+import torch.optim as optim
+
+
+def fanin_init(m):
+    fan_in, _ = nn.init._calculate_fan_in_and_fan_out(m.weight)
+    bound = 1.0 / math.sqrt(fan_in)
+    nn.init.uniform_(m.weight, -bound, bound)
+    nn.init.uniform_(m.bias, -bound, bound)
+
+
+def uniform_init(m, bound):
+    nn.init.uniform_(m.weight, -bound, bound)
+    nn.init.uniform_(m.bias, -bound, bound)
+
+
+class ContinuousDeterministic(Model):
```

Comment:
Better class name. It's not easy to tell that this class is a Model class just from its name right now.